src/Discord.Net.WebSocket/DiscordSocketClient.Events.cs

```diff
@@ -59,12 +59,12 @@ public partial class DiscordSocketClient
             remove { _messageReceivedEvent.Remove(value); }
         }
         private readonly AsyncEvent<Func<SocketMessage, Task>> _messageReceivedEvent = new AsyncEvent<Func<SocketMessage, Task>>();
-        public event Func<ulong, Optional<SocketMessage>, Task> MessageDeleted
+        public event Func<ulong, SocketChannel, Optional<SocketMessage>, Task> MessageDeleted
```

Comment:
Shouldn't this be `ISocketMessageChannel` instead of `SocketChannel`? Other events that receive a channel type object receive the interface not an object.